apps/OboeTester/app/src/main/java/com/mobileer/oboetester/MicrophoneInfoConverter.java

```diff
@@ -60,6 +60,9 @@ public static String reportMicrophoneInfo(MicrophoneInfo micInfo) {
         sb.append("\nType       : " + micInfo.getType());
 
         List<Pair<Integer, Integer>> mapping = micInfo.getChannelMapping();
+        if (mapping == null) {
```

Comment:
Are we sure null is an error? What about if this is empty?

The documentation for the function seems a bit weird https://developer.android.com/reference/android/media/MicrophoneInfo#getChannelMapping()

Reply:
Jiabin says yes, it is a regression.